src/lib/y2partitioner/icons.rb

```diff
@@ -56,6 +56,8 @@ module Icons
     # icon
     BTRFS = LVM
 
+    TMPFS = BTRFS
+
```

Comment:
Reusing the LVM icon for tmpfs doesn't look like a great idea. The current icon consist on several stacked hard disks, which makes a lot of sense for LVM but definitely not for tmpfs.

Reply:
You are right. Maybe this? https://github.com/libyui/libyui-qt/blob/master/src/icons/drive-partition.svg